department-of-veterans-affairs/caseflow
# CASEFLOW-376 (Part 1) Allow optional dates for straight_reversal and death_dismissal

Connects #15820 (Part 1 of [CASEFLOW-376](https://vajira.max.gov/browse/CASEFLOW-376))

Reason: So that CASEFLOW-376 can be demo'd.

### Description
For straight_reversal and death_dismissal types of CAVC Remands, allow judgement & mandate dates to be optional.

### Acceptance Criteria
- [ ] For straight_reversal and death_dismissal, judgement & mandate dates are optional when filling out the form.

### Testing Plan
(See Testing Plan in #15774 for more detailed instructions.)
1. Fill out a CAVC Remand form, picking either straight_reversal or death_dismissal.
2. Submit form without judgement & mandate dates
3. Check that resulting appeal is created
